instructionAPI/src/InstructionDecoder-amdgpu-vega.C

```diff
@@ -137,7 +138,7 @@ namespace Dyninst {
 		void InstructionDecoder_amdgpu_vega::makeBranchTarget(bool branchIsCall, bool bIsConditional, int immVal,
 				int immLen = 16) {
 			Expression::Ptr lhs = makeAddExpression(makePCExpr(),Immediate::makeImmediate(Result(s48,4)),s48);
-			int64_t offset = sign_extend64(16, immVal * 4);
+			int64_t offset = sign_extend64(immLen, immVal * 4);
```

Comment:
Could we move this change to its own PR? It's a good change, but not in the scope of the compiler warning fixes.

Reply:
It was complaining about the non-used function parameter immLen, which made me realize that this why I included immLen in the first place. I'll revert the change and remove the parameter name.

Reply:
Oh, I see. Let's use the passed parameter as you fixed it.

Reply:
This change is kept.